lib/l10n/app_en.arb

```diff
@@ -179,7 +180,7 @@
     "map_page_location_permission_go_to_settings": "Turn On in Settings",
     "map_page_location_permission_reasoning_settings": "Plante works best with Location Services turned on. You'll get information about stores based on your location",
     "map_page_location_permission_title": "Location Services",
-    "map_page_no_shops_hint2": "No stores with products nearby. You can tune stores filters by clicking",
+    "map_page_no_shops_hint2": "There are no stores in the area yet. Try moving the map, add a new store or try the map filters.",
```

Comment:
The string ID probably should be changed to `map_page_no_shops_hint3` since new translations will 100% be needed :)